tox.ini

```diff
@@ -16,14 +17,17 @@ envlist = lint
 [testenv]
 usedevelop = True
 install_command = pip install -U {opts} {packages}
+passenv =
+    PY_COLORS
+    NO_COLORS
+    TERM
 setenv =
     VIRTUAL_ENV = {envdir}
-commands_pre =
-    {toxinidir}/utils/detox {envdir}
-envars =
-    PATH = {env:PATH}:{toxinidir}/bin
+    XDG_CACHE_HOME = {envdir}
```

Comment:
I am just curious, how do we use tell `black` to store its cache in the path defined by this variable `XDG_CACHE_HOME`

Reply:
See https://black.readthedocs.io/en/stable/ignoring_unmodified_files.html.